### PR TITLE
feat: configurable view directory

### PR DESCRIPTION
**Pull request by XadillaX**

### CODE REVIEW

**Comment by fengmk2** [comment]
commit log don't use Chinese.

File: config/index.js [hunk 1]
```diff
@@ -47,6 +47,9 @@ var config = {
   // web page viewCache
   viewCache: false,
 
+  // theme files directory
+  themeDir: path.join(root, 'view', 'web'),
```
Comment (fengmk2) [inline config/index.js]: viewDir is better than themeDir

Comment (XadillaX) [reply config/index.js]: done.

**Comment by fengmk2** [comment]
Release 3.0.0-rc.7

**Pull request merged**